packages/thirdweb/src/exports/react.ts

```diff
@@ -178,4 +178,6 @@ export { useConnectionManager } from "../react/core/providers/connection-manager
  */
 export { Blobbie } from "../react/web/ui/ConnectWallet/Blobbie.js";
 
+export { Avatar } from "../react/web/ui/ConnectWallet/Avatar.js";
+
```

Comment:
We should probably name this component `ENSAvatar` or maybe follow the provider pattern and have `ENS`, `ENS.Avatar`, `ENS.Name` components ?

Reply:
If it's for displaying both ENS avatar and non-ENS avatar, do we want to name it "ENS"?